### PR TITLE
🤖 TinaGPT chat-bot 

### PR DESCRIPTION
**Pull request by joshbermanssw**
### General Contributing:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tinacms.org/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
As per [#1748](https://github.com/tinacms/tina.io/issues/1748) I have created a ChatBot using ChatBase and have implemented this to the main layout of [Tina.io](Tina.io)

Screenshot 2024-05-06 at 10 39 44 am
**Figure: 🏡 Home page view with chat-bot icon on bottom right**

Screenshot 2024-05-06 at 10 39 30 am
**Figure: 🦙Speaking to TinaGPT on local version of Tina.io site**

### All New Content Submissions: (To be confirmed by reviewer)

* [ ] Title is short & specific
* [ ] Headers are logically ordered & consistent
* [ ] Purpose of document is explained in the first paragraph
* [ ] Procedures are tested and work
* [ ] Any technical concepts are explained or linked to
* [ ] Document follows structure from templates
* [ ] All links work
* [ ] The spelling and grammar checker has been run
* [ ] Graphics and images are clear and useful
* [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: components/ui/TinaChatBot.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Script from 'next/script'
+
+const chatBaseBotId = 'mJl22IM4pXzWC60JoM5Fj'
```
Comment (amankumarrr) [inline components/ui/TinaChatBot.tsx:3]: I think we should reference this from .env, (i.e process.env.CHATBASE_BOT_ID)

File: components/ui/TinaChatBot.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Script from 'next/script'
+const chatBaseBotId = process.env.NEXT_PUBLIC_CHATBASE_BOT_ID
```
Comment (JackDevAU) [inline components/ui/TinaChatBot.tsx:2]: Have we added this to the env already 👀

**Review by JackDevAU — APPROVED** [review]
🎉

File: components/ui/TinaChatBot.tsx (hunk 2 above)
Comment (joshbermanssw) [reply components/ui/TinaChatBot.tsx:2]: 👍

**Pull request merged**